external/java/src/main/java/ch/epfl/dedis/lib/omniledger/InstanceId.java

```diff
@@ -69,7 +70,18 @@ public String toString(){
         return DatatypeConverter.printHexBinary(id);
     }
 
-    public ByteString toProto(){
+    public ByteString toByteString(){
         return ByteString.copyFrom(id);
     }
+
+    public TransactionProto.ObjectID toProto(){
```

Comment:
this is a bit confusing to me, so we don't have an `ObjectID` class anymore, but the equivalent in java is called `InstanceID`?